### PR TITLE
EHN: Add ZFrameRegistration extension. 

### PR DESCRIPTION
**Pull request by leochan2009**
Description: This cli module takes an ZFrame MRI image as input, calculates the zframe transformation matrix in the RAS coordinates.

### CODE REVIEW

**Comment by leochan2009** [comment]
@lassoan 

Hi Andras,
Thanks for your comments. 
I modified the extension according to your kind suggestions:
- [x] In https://github.com/leochan2009/ZFrameCalibration/blob/master/CMakeLists.txt, add valid images for icons and screenshots
- [x] In https://github.com/leochan2009/ZFrameCalibration/blob/master/README.md, add one-paragraph summary, sample screenshots, example data set (you can upload to MIDAS, if you are not sure how to do it I can help), and basic tutorial about how a user can try this module. Specify this page as documentation URL.
- [x] Consider using a more accurate module name: in itself, this module performs a registration operation, that an application may use for calibration. It would be more accurate to be called ZFrameRegistration.
 Extension repository name should start with Slicer (SlicerZFrameRegistration, the extension name would be still just ZFrameRegistration).

**Comment by pieper** [comment]
👍

**Comment by leochan2009** [comment]
@pieper @lassoan 
Hi,
I just finished the necessary steps, tests added, user manual added, please see the link:
https://github.com/leochan2009/SlicerZFrameRegistration/blob/master/README.md
what else should i do to be able to publish the extension?
Thanks in advance,
Best,
Longquan

**Comment by leochan2009** [comment]
The branch is transfered from private into SlicerProstate.

File: ZFrameRegistration.s4ext [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
+# First token of each non-comment line is the keyword and the rest of the line
+# (including spaces) is the value.
+# - the value can be blank
+#
+
+# This is source code manager (i.e. svn)
+scm git
+scmurl https://github.com/leochan2009/SlicerZFrameRegistration.git
```
Comment (jcfr) [inline ZFrameRegistration.s4ext]: Could you update to SlicerProstate/SlicerZFrameRegistration ?

**Comment by jcfr** [comment]
Closing. Superseded by https://github.com/Slicer/ExtensionsIndex/pull/1476 which was generated using:

```
cd /tmp
git clone git@github.com:SlicerProstate/SlicerZFrameRegistration.git
cd SlicerZFrameRegistration/
~/Projects/Slicer-2-build/Slicer-build/bin/slicerExtensionWizard --contribute .
```

**Comment by leochan2009** [comment]
Thanks !

File: ZFrameRegistration.s4ext (hunk 1 above)
Comment (leochan2009) [reply ZFrameRegistration.s4ext]: Sure

Comment (jcfr) [reply ZFrameRegistration.s4ext]: FYI, this is not need anymore. A new Pr has been submitted to update the extension. See below

**Pull request closed without merging**